src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationOptions.cs

```diff
@@ -226,116 +185,49 @@ public OpenIdConnectProtocolValidator ProtocolValidator
         /// <summary>
         /// Gets or sets the 'response_mode'.
         /// </summary>
-        public string ResponseMode { get; private set; }
+        public string ResponseMode { get; private set; } = OpenIdConnectResponseModes.FormPost;
 
         /// <summary>
         /// Gets or sets the 'response_type'.
         /// </summary>
-        public string ResponseType { get; set; }
+        public string ResponseType { get; set; } = OpenIdConnectResponseTypes.CodeIdToken;
 
         /// <summary>
-        /// Gets or sets the 'scope'.
+        /// Gets the list of permissions to request.
         /// </summary>
-        public string Scope { get; set; }
```

Comment:
Replaced by a list for consistency: https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNet.Authentication.OAuth/OAuthAuthenticationOptions.cs#L92